src/shared/components/data-table/components/table-controls/components/table-actions/components/download-table-data/DownloadTableData.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+    - Download has two variations
+        - Download shown data: https://xd.adobe.com/view/78773ed6-d738-4ea6-be84-fcc73487eac4-2d24/screen/24f1cdb0-360c-46fc-99c4-7218d3a0ab51?fullscreen
+        - Download all data: https://xd.adobe.com/view/78773ed6-d738-4ea6-be84-fcc73487eac4-2d24/screen/3a7efbd5-3040-4f72-9f31-f7d1c496658e?fullscreen
+    
+*/
```

Comment:
This is the same as `TableFilter`. Do we need this to be in this PR?

Reply:
This can go. I've removed it.